app/apollo/resolvers/cluster.js

```diff
@@ -183,6 +183,65 @@ const clusterResolvers = {
       return results;
     }, // end clusterCountByKubeVersion
   }, // end query
+
+  Mutation: {
+    deleteClusterByClusterID: async (
+      parent,
+      { org_id, cluster_id },
+      context,
+    ) => {
+      const queryName = 'deleteClusterByClusterID';
+      const { models, me, req_id, logger } = context;
+      logger.debug({req_id, user: whoIs(me), org_id, cluster_id}, `${queryName} enter`);
+
+      await validAuth(me, org_id, ACTIONS.MANAGE, TYPES.CLUSTER, queryName, context);
+
+      try {
+        const deletedCluster = await models.Cluster.findOneAndDelete({org_id,
+          cluster_id});
+
+        const deletedResources = await models.Resource.updateMany({ org_id, cluster_id }, 
```

Comment:
Delete the resources from the database (not soft delete)

Reply:
per discussion with mike, keep soft delete for now until we have a background process to remove contents from COS.